python/dgl/model_zoo/chem/README.md

```diff
@@ -40,10 +40,16 @@ mostly developed based on molecule fingerprints.
 Graph neural networks make it possible for a data-driven representation of molecules out of the atoms, bonds and 
```

Comment:
Synchronize for the update you will make in `examples/pytorch/model_zoo/chem/property_prediction/README.md`.